go.mod

```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-12
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.3-0.20220307013541-e3c38bb961a0
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-12...e3c38bb961a0?expand=1

Reply:
I remember you have said that doing it in checkTx won't help.

Can you describe a bit more about this solution?